Dockerfile

```diff
@@ -27,6 +27,8 @@ FROM $BASE_IMAGE AS builder
 # Copy source
 COPY . .
 
+ENV PY_VER=python3.8
```

Comment:
Why 3.8 instead of newer such as 3.10+ ?

Reply:
Since #252 was merged, we require python 3.10 in-practice anyway (in terms of what's shipped by the image our docker images are based upon). I'm quite comfortable pushing this to 3.10.